lib/spack/docs/containers.rst

```diff
@@ -179,7 +190,211 @@ of environments:
          app: "gromacs"
          mpi: "mpich"
 
-The tables below describe the configuration options that are currently supported:
+A detailed description of the options available can be found in the
+:ref:`container_config_options` section.
+
+-------------------
+Setting Base Images
+-------------------
+
+The ``images`` subsection is used to select both the image where
+Spack builds the software and the image where the built software
+is installed. This attribute can be set in two different ways and
+which one to use depends on the use case at hand.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Use Official Spack Images From Dockerhub
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To generate a recipe that use an official Docker image from the
+Spack organization to build the software and the corresponding official OS image
+to install the built software, the user is just requested to
+specify:
+
+1. An operating system under ``images:os``
+2. A Spack version under ``images:spack``
+
+Any combination of these two values that can be mapped to one of the images
+discussed in :ref:`container_spack_images` is allowed. For instance, the
+following ``spack.yaml``:
+
+.. code-block:: yaml
+
+   spack:
+     specs:
+     - gromacs+mpi
+     - mpich
+
+     container:
+       images:
+         os: centos/7
+         spack: 0.15.4
+
+uses ``spack/centos7:0.15.4``  and ``centos:7`` for the stages where the
```

Comment:
This appears to address https://github.com/spack/spack/pull/15028#discussion_r451242531

Reply:
Yes, that was the intent.